pom.xml

```diff
@@ -44,7 +43,7 @@
  
         <module>org.eclipse.lyo.tools.updatesite</module>
  
-        <module>org.eclipse.lyo.oslc4j.plugins</module>
+        <!-- <module>org.eclipse.lyo.oslc4j.plugins</module> -->
```

Comment:
I guess this is the only part left to fix?

Reply:
Yes. What is strange, is that I can run "mvn package" within the plugins project. Things work fine then.
So one solution is to make 2 builds with 2 pom.xml files. One for LyoDesigner, the other for plugins.
Otherwise, I seem to get conflicts with the build/maven/tycho plugins.

Reply:
Fine with me if you want to keep those as 2 different parent POMs, but I guess those plugins are published as a p2 site and not on maven so it may be worth to figure out the root cause.

Reply:
@berezovskyi  See my analysis below, in last comment. 
I can make it build all together, but not sure how neat the resulting structure is.